hmpps-prisoner-search-indexer/helm_deploy/hmpps-prisoner-search-indexer/templates/index-queues-prometheus-rules.yaml

```diff
@@ -23,7 +23,7 @@ spec:
             {{ $businessOrAllHoursExpression }}
           for: 3h
           labels:
-            severity: syscon
+            severity: {{ index .Values "generic-prometheus-alerts" "alertSeverity" }}
```

Comment:
have to use strange `index` syntax to get the severity variable out since there are hyphens in `generic-prometheus-alerts`